### PR TITLE
Fix background color on communities/chat home

### LINKED ISSUE #19518: Wrong background colour on Community / Messaging screens in dark mode [IOS]
Reproducible on IOS only.

**Steps:** 

1. Enable dark mode
2. Navigate to Community or Messaging screens
3. Pay attention at the background colour of header

**Expected result:** 

https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=5040-188678&mode=design&t=wMCVUv212R5saUvR-0

**Actual result:**  header background  differs from the expected 

photo_2024-04-04 09 36 55
photo_2024-04-04 09 36 59
photo_2024-04-04 09 37 03

### Additional Information

- Status version: nightly 
- Operating System:  iOS 16.4.7
- Device: iPhone X

[comment]: # (Please, add logs/notes if necessary)

### PR DESCRIPTION
**Pull request by ibrkhalil**
fixes #19518 

image
Fixes an issue with background color on dark mode on iOS on communities/chats home screen

status: ready

### CODE REVIEW

File: src/status_im/common/home/banner/style.cljs [hunk 1]
```diff
@@ -26,24 +27,26 @@
...
-    :position :absolute
-    :top      0
-    :right    0
-    :left     0
-    :height   (+ (safe-area/get-top) 244)}))
+   {:overflow         (if platform/ios? :visible :hidden)
+    :z-index          1
+    :position         :absolute
+    :top              0
+    :background-color (colors/theme-colors colors/white colors/neutral-95)
```
Comment (mohsen-ghafouri) [inline src/status_im/common/home/banner/style.cljs]: shouldn't we pass `theme` here?

```suggestion
    :background-color (colors/theme-colors colors/white colors/neutral-95 theme)
```

Comment (J-Son89) [reply src/status_im/common/home/banner/style.cljs]: same for the other uses in this pr 👍

Comment (ibrkhalil) [reply src/status_im/common/home/banner/style.cljs]: Added, Thanks guys!

**Review by mohsen-ghafouri — APPROVED** [review]
LGTM

**Comment by VolodLytvynenko** [comment]
Hi @ibrkhalil thank you for PR. Ready to be merged

**Pull request merged**